components/Select/src/index.scss

```diff
@@ -4,6 +4,11 @@
   box-sizing: var(--denhaag-select-box-sizing);
   max-block-size: var(--denhaag-select-max-block-size);
 
+  &.utrecht-select {
```

Comment:
Moet hij niet getoond worden als je alleen eenhaag-select gebruikt?

Reply:
De select krijg altijd deze classes nu: `utrecht-select utrecht-select--html-select denhaag-select`. Wanneer je alleen de selector `.denhaag-select` gebruikt, wordt de background position overschreven door verkeerde (en niet instelbare) van Utrecht.

`.denhaag-select.utrecht-select` gebruik ik omdat deze specifieker is en dus niet overschreven wordt.

In de ideale situatie maakt Utrecht die background-position instelbaar met een design token, maar omdat we dat nu niet zelf in de hand hebben en de samenwerking met Utrecht ook matig is op dit moment, heb ik het voor nu zo opgelost.